index.html

```diff
@@ -6,11 +6,10 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/public/favicon-32x32.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>Loopstudios - Immersive VR Experiences for Leading Brands</title>
   </head>
   <body>
     <div id="root"></div>
```

Comment:
Maybe we could change the `div` element to `main` so it is more semantically correct?
```suggestion
    <main id="root"></main>
```